### DISCUSSION TITLE
Any suggestions regarding to OM-android app contribution?

### DISCUSSION THREAD

**Discussion by LaachirDeeper** [organicmaps/organicmaps#6548] [Q&A]
Hello guys, Hop everyone doing great.

Yesterday I just downloaded GitHub app on android to see how OM tracking etc will be on it, But I like it, it looks awesome, so that I can be close much to OM community to understand its development code etc :)

The OM becomes my main map app since MapsMe becomes like a rubbish by a tones of irrelevant map features. I just complete a 7000Km eu cycle trip using completely OM in it, keep up the good work, and the most importantly is users privacy-care

**What you guys suggest to me regarding to the contribution to OM-android app, I mean any tools, ideas, softwares etc. Because it's my first time on code contributions via GitHub, One thing that I strangle with is Importing the code into Android Studio, I've getting a lot of non-understandable errors because its contains code of IOS and many unnecessary stuff as an android developer.**

So yeah, any suggestions, ideas please let me know. Thank you!

This is my cycling finished trip, and I always use OM beta version.
![Screenshot_2023-11-12-09-38-12-86.jpg](https://github.com/organicmaps/organicmaps/assets/25303041/01f1abc8-1a0a-4e04-823e-f2d784aac520)

**Comment by cyber-toad** [comment]
The main instruction with details how to build OM from source is available here: https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md

**Reply by LaachirDeeper** [reply]
I already saw this.. If there's any other suggestions, ideas etc. :)

**Reply by biodranik** [reply]
Thanks for using OM! 
Are you importing the root folder or the android folder to Android Studio? The latter is correct. 

Did you follow the instructions?

**Reply by LaachirDeeper** [reply]
I was importing it by the link.., but as I saw on the instructions it's better to import it by downloading the OM to the computer and then access it directly through. I will give you update...
